### ISSUE TITLE
Add isHttpPost similar to isEmail in templates

### ISSUE THREAD

**Issue by gbrayut** [bosun-monitor/bosun#786]
We have a few alerts now that add a bunch of links when posting to chat. Right now they use the isEmail to to keep the email subjects clean, however this still causes issues on the dashboard:

![image](https://cloud.githubusercontent.com/assets/304401/6630295/e63d5726-c8dc-11e4-9d55-81870a31a54c.png)

adding isHttpPost support to templates means we could put the links in there behind an if statement so the dashboard still shows up normal.

**Comment by kylebrandt** [comment]
We talked about this. Right now I think the best idea might be to have additional optional parameters in the template definition. So subject and body still exist, but we could also have an dashboard_subject, http_subject etc to the template def. If those are not specified than it falls back to using the the subject and body specified for email.

"No dashboard" will be used as part of the log thing @mjibson is working on, so that might be relevant here as well in part of the ...? template definition... I think.

Thoughts?

**Comment by mjibson** [comment]
I do not agree that's the best idea. I think adding more of these target rendering things is going to get messy quickly. I'm more in favor of expanding the existing template overwriting to be better. That is, the alert template is a simple as possible, and downstream notifications can modify the templates as they see fit.
